test-resources/linting_formatters_example.clj

```diff
@@ -0,0 +1,49 @@
+(ns linting-formatters-example
+  "Documents how formatters can be used as linters for a non-mutative experience.
+
+  Not a part of the test suite.
+
+  NOTE: Highly subject to change! see https://github.com/nedap/formatting-stack/pull/152
```

Comment:
I cannot clearly see the connection between #152 and the warning itself as `overrides` is not used

Reply:
#152 introduces `:overrides`, so it cannot be used. That's the point.


Once `:overrides` is merged, this example shall be updated to use it.

Reply:
Ah, it was not merged yet! It was that old, I expected that it was merged.